docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,23 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
+
+> In interfaces, functions in actual declarations can be non-abstract and have a
+> body. To indicate that in `expect` declarations, explicitly mark functions as `open`:
+>
+> ```kotlin
+> expect interface Mascot {
```

Comment:
Please, add the usage of this interface in common code, otherwise it's completely not clear why this feature is even needed. Example:

```kotlin
expect interface Mascot {
    open fun name() 
}

class MascotImpl : Mascot {
    // fine not to implement 'name': all 'actual's are guaranteed to have a default implementation for it
}
```